scripts/release.sh

```diff
@@ -0,0 +1,278 @@
+    git merge $develop_branch $main_branch
+fi
+
+echo
+# 1 Create new release for tag
+prompt_user "Would you like to open to the Github releases page to create a release?"
+if [[ $? == 1 ]]; then
+    open "https://github.com/smartdevicelink/sdl_ios/releases"
+
+    # TODO - phase 4 - this can be automated with gh
+    # https://cli.github.com/manual/gh_release_create
+    # TODO - can we pull the list of changes from Changelog.md and automatically add those to the release (so we do not type the same things twice)
+    # TODO - if/when we automate this, make sure to open the releases page so the user can review it.
+fi
+
+echo
+# 2 push new release to primary cocoapod
+# 3 Push the new release to the secondary cocoapod using command line:
+prompt_user "Would you like to push the release to CocoaPods?"
+if [[ $? == 1 ]]; then
+    pod trunk push SmartDeviceLink.podspec --allow-warnings
+    pod trunk push SmartDeviceLink-iOS.podspec --allow-warnings.
+fi
+
+# 4 Add a binary xcframework archive for manual installation with the following commands
+echo
+prompt_user "Would you like to create a binary xcframework for manual installation?"
+if [[ $? == 1 ]]; then
+    xcodebuild archive -project 'SmartDeviceLink-iOS.xcodeproj/' -scheme 'SmartDeviceLink' -configuration Release -destination 'generic/platform=iOS' -archivePath './SmartDeviceLink-Device.xcarchive' SKIP_INSTALL=NO
+    xcodebuild archive -project 'SmartDeviceLink-iOS.xcodeproj/' -scheme 'SmartDeviceLink' -configuration Release -destination 'generic/platform=iOS Simulator' -archivePath './SmartDeviceLink-Simulator.xcarchive' SKIP_INSTALL=NO
+    xcodebuild -create-xcframework -framework './SmartDeviceLink-Device.xcarchive/Products/Library/Frameworks/SmartDeviceLink.framework/' -framework './SmartDeviceLink-Simulator.xcarchive/Products/Library/Frameworks/SmartDeviceLink.framework/' -output './SmartDeviceLink.xcframework'
+
+    folder="SmartDeviceLink.xcframework"
+    zip_file_name="SmartDeviceLink-$new_version_number.xcframework.zip"
+    if [ -f $zip_file_name ]; then
+        #kill the old zip if present.  Useful for re-running the script
+        rm $zip_file_name 
+    fi
+    #verify file exists before acting on it.
+    if [ -d "$folder" ]; then
+        zip $zip_file_name $folder
+    fi
+    #Check to see if the zip exists, and then remove old files.
+    if [ -f "$zip_file_name" ]; then
+        rm -r $folder
+    fi
+    echo
+    echo "The xcframework zip file was created at $zip_file_name. Please add it to the Github Release, then press enter..."
+    read user_input
+    #TODO - phase 4 - automate adding to release
+fi
+
+# Rename the docset and pack it
+# SmartDeviceLink-$new_version_number-docset.tgz
+docset_directory="docs/docsets/"
+docset_tar_file_name="SmartDeviceLink-$new_version_number-docset.tgz"
+tar -czf $docset_tar_file_name $docset_directory
+echo 
+echo "Please add the docset at $docset_tar_file_name to the Github release, then press enter..."
+read user_input
```

Comment:
1. Prompt user if they'd like to create the docset
2. Move the zipped file to the main folder

Reply:
Prompt has been added.  What do we mean by "main folder".  In my testing the zip is in the sdl_ios root